### PR TITLE
Tidy HTML structure

### PR DESCRIPTION
**Pull request by AnnaUlyanova**
Hey,

Did some formatting & indenting the HTML, based on advices from this link - https://www.granneman.com/webdev/coding/formatting-and-indenting-your-html/.
Please review :) thanks

### CODE REVIEW

File: pristine-zen-garden.html [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+            <p>
+              Littering a dark and dreary road lay the past relics of browser-specific tags, incompatible <abbr title="Document Object Model">DOM</abbr>s, broken <abbr title="Cascading Style Sheets">CSS</abbr> support, and abandoned browsers.
+            </p>
+            <p>
+              We must clear the mind of the past. Web enlightenment has been achieved thanks to the tireless efforts of folk like the <abbr title="World Wide Web Consortium">W3C</abbr>, <abbr title="Web Standards Project">WaSP</abbr>, and the major browser creators.
+            </p>
+            <p>
+              The CSS Zen Garden invites you to relax and meditate on the important lessons of the masters. Begin to see with clarity. Learn to use the time-honored techniques in new and invigorating fashion. Become one with the web.
+            </p>
+    </div>
```
Comment (JessicaNeary) [inline pristine-zen-garden.html]: this </div> tag needs to be indented more

File: pristine-zen-garden.html [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+            </p>
+            <p>
+              This is a learning exercise as well as a demonstration. You retain full copyright on your graphics (with limited exceptions, see <a href="http://www.mezzoblue.com/zengarden/submit/guidelines/">submission guidelines</a>), but we ask you release your <abbr title="Cascading Style Sheets">CSS</abbr> under a Creative Commons license identical to the <a href="http://creativecommons.org/licenses/by-nc-sa/3.0/" title="View the Zen Garden's license information.">one on this site</a> so that others may learn from your work.
+            </p>
+            <p role="contentinfo">By <a href="http://www.mezzoblue.com/">Dave Shea</a>.
+              Bandwidth graciously donated by <a href="http://www.mediatemple.net/">mediatemple</a>. Now available: <a href="http://www.amazon.com/exec/obidos/ASIN/0321303474/mezzoblue-20/">Zen Garden, the book</a>.
+            </p>
+      </div>
+
+  <footer>
```
Comment (JessicaNeary) [inline pristine-zen-garden.html]: Footer needs to be indented one more space

File: pristine-zen-garden.html [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+            <div class="zen-resources" id="zen-resources">
+                <h3 class="resources">Resources:</h3>
+                  <ul>
+                        <li class="view-css">
+                          <a href="/214/214.css" title="View the source CSS file of the currently-viewed design.">
+                        View This Design&#8217;s <abbr title="Cascading Style Sheets">CSS</abbr></a>
+                        </li>
+                        <li class="css-resources">
+                          <a href="http://www.mezzoblue.com/zengarden/resources/" title="Links to great sites with information on using CSS.">
+                        <abbr title="Cascading Style Sheets">CSS</abbr> Resources</a>
```
Comment (JessicaNeary) [inline pristine-zen-garden.html]: Second line of <a href> isn't in line with the start of the tag

**Review by JessicaNeary — CHANGES_REQUESTED** [review]
Pretty good indention, just a couple lines that need adjusting. Adding more whitespace between items might be a good idea to make the code a bit clearer too.

File: pristine-zen-garden.html (hunk 3 above)
Comment (AnnaUlyanova) [reply pristine-zen-garden.html]: Hi Jessica, thanks for the review. I've corrected the indention followed all the comments. Please review again :) Thanks heaps!

**Pull request merged**